test/camelCaseLocalFunctionsRule/camelCaseLocalFunctionsRule.tsx.fix

```diff
@@ -0,0 +1,26 @@
+import lowerImport from 'lower';
+import UpperImport from 'upper';
+
+function lower() { }
+function Upper() { }
+
+const aliasedUpper = Upper;
+
+const obj = {
+  lower() { },
+  Upper() { }
+};
+
+lower();
+lower(42);
+aliasedUpper();
+lowerImport();
+Upper.bind(null);
+obj.lower();
+obj.Upper();
+const jsx1 = <Upper />;
+const jsx2 = <UpperImport />;
+
+const fails1 = upper();
+const fails2 = upperImport();

```

Comment:
Looks like this autofix will generate code that doesn't compile 100% of the time since it doesn't rename by symbol, so it could be argued it's making things worse (style rule violation -> compiler error). If you trust the autofixes, which is arguably the goal here, this would be an odd one out that needs to be cleaned up after. Let's drop it until we can do a symbol rename?